configuration/transitions.md

```diff
@@ -57,17 +57,18 @@ Understanding the different types of reports used in the configuration:
              |           latest_report comes in, suspected_cholera alert is sent
              |           |
              v           v
----[---*-o---*--*--o-o---*]------->    time
+---[---o-o---o--o--o-o---o]------->    time
+       *     *  *        *
                 1        0
 ```
 
-`[]` : time window
+`[]` is the time window
 
-`*` and `o` : `reports` : any report that came in to the server.
+`o` markers are `reports` : any report that came in to the server.
 
-`*` : `counted_reports` : reports that came in that passed the `is_report_counted` filter function.
+`*` markers are `counted_reports` : reports that came in that passed the `is_report_counted` filter function.
```

Comment:
IMO change to:
> that passed the configured `is_report_counted` filter function

Because to me that sounds like you're talking about the transition filter. Might just be dev perspective